src/tutorials/race_timer.md

```diff
@@ -0,0 +1,4383 @@
+    -- The function will return (exit) if there are 0 players to prevent
+    -- statements from executing further down in the body of the function.
+
+    local function task_handler()
+        if raceTask == nil then
+            raceTask = Task.Spawn(task_handler, 14)
+
+            -- Change "repeatInterval" if the race should last longer.
+            -- This task handles the game state in a very simplified way.
+
+            raceTask.repeatInterval = 10
+            raceTask.repeatCount = -1
+        end
+
+        if #Game.GetPlayers() == 0 then
+            return
+        end
+
+        StopRace()
+
+        Task.Wait(2)
+
+        MovePlayersToStart()
+        TellPlayersToGetReady()
+
+        Task.Wait(2)
+
+        TellPlayersToGo()
+
+        EnablePlayers()
+    end
+
+    task_handler()
+    ```
+
+### Update Race Manager Client Script
+
+We need to update the **Race_Manager_Client** script so that it has references to a few things as well as update the UI with the split times.
+
+1. Drag and drop the **Splits** panel onto the script as a custom property and rename it to **splitsPanel**.
+2. Drag and drop the **Splits** group inside the **Triggers** group onto the script as a custom property and rename it to **splitTriggers**.
+3. Drag and drop the template for the split entry onto the script and rename it to **splitEntry**.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/RaceTimerTutorial/split_client_props.mp4" type="video/mp4" />
+    </video>
+</div>
+
+We now need to update the **Race_Manager_Client** script.
+
+```lua linenums="1"
+local SPLITS_PANEL = script:GetCustomProperty("splitsPanel"):WaitForObject()
+local SPLIT_TRIGGERS = script:GetCustomProperty("splitTriggers"):WaitForObject()
+local SPLIT_ENTRY = script:GetCustomProperty("splitEntry")
+```
+
+Add the above code to the top of the script just below the `LAST_TIME` variable.
+
+```lua linenums="1"
+local splits = {}
+local currentSplit = 0
+```
+
+Add the above code just below the variable `localPlayer`.
+
+- `splits`
+
+    This table will get updated with the split the player has overlapped so we can keep track of them through the race.
+
+- `currentSplit`
+
+    This will be updated with the current child index of the split inside the **Splits** panel.
+
+```lua linenums="1"
+local function ClearActiveSplit()
+    if(currentSplit > 0) then
+        splits[currentSplit].raceTime:SetColor(Color.WHITE)
+        splits[currentSplit].splitTime:SetColor(Color.WHITE)
+        splits[currentSplit].splitName:SetColor(Color.WHITE)
+    end
+end
+```
+
+Add the above function just below `Tick` function. This function will clear the current active split by setting the color of each text object to white. It makes sure the current split is greater than `0` as array indexing in `Lua` starts from `1`.
```

Comment:
"This function" -> state the name of the function in Code format

Replace "It" with the name of the function.